workflows/creating-a-workflow.md

```diff
@@ -1,12 +1,12 @@
 # Creating a Workflow
 
 {% hint style="info" %}
-This guide assumes you've already created an Aqueduct API client. If you haven't done that, please see the [quickstart-guide.md](../quickstart-guide.md "mention") for more details.
+This guide assumes you've already created an Aqueduct API client. If you haven't done that, please see the [quickstart guide](../quickstart-guide.md "mention") for more details.
```

Comment:
What does "mention" do in markdown? Also, I don't think the quickstart guide includes information on Aqueduct API client creation. That is the snippet you copy paste from the account section, right?

Reply:
This quick start guide here refers to the actual Aqueduct quick start guide https://docs.aqueducthq.com/quickstart-guide. The only change I made on this line is to have the link show 'quickstart guide' instead of 'quickstart-guide.md'.

Reply:
I have no idea what "mention" do, I just copy it from elsewhere. (@vsreekanti do you know?)

Reply:
Oh, maybe we should update the Quickstart Guide to have a little more descriptive name since we have an Integration Management Quickstart Guide and this one.

Reply:
I actually follow some other example like https://docs.ray.io/en/latest/ , and I don't feel it's strongly necessary to add more description to the 'main' quickstart guide's name. It seems to be a common pattern that there's a overall quickstart and separate ones for each smaller component. So I think as long as we keep the structure clear, users should be able to distinguish them while reading

Reply:
I wouldn't really call the integration management guide a quickstart. It isn't particularly quick. 🙂 I would instead just call it an integration management overview maybe.

Reply:
Let's reserve the word quickstart for our main guide.